### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by DannyS03**
General updates to avail light readme

### CODE REVIEW

**Review by prabal-banerjee — APPROVED** [review]
Minor changes in the intro is expected by end of week. Should we change it in this PR, you review and then we merge?

**Comment by DannyS03** [comment]
> Minor changes in the intro is expected by end of week. Should we change it in this PR, you review and then we merge?

That works, the only obstacle is having access to the branch on your end

File: README.md [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
 
-Light client for Data Availability Blockchain of Polygon 💻
+[![Build status](https://github.com/maticnetwork/avail-light/actions/workflows/default.yml/badge.svg)](https://github.com/maticnetwork/avail-light/actions/workflows/default.yml) [![Code coverage](https://codecov.io/gh/maticnetwork/avail-light/branch/main/graph/badge.svg?token=7O2EA7QMC2)](https://codecov.io/gh/maticnetwork/avail-light)
 
 ![demo](./img/prod_demo.png)
 
 ## Introduction
 
-Naive approach for building one DA light client, which will do following
+Build a data availability light client that will do following:
```
Comment (vthunder) [inline README.md:14]: I'd suggest something like: `avail-light` is a data availability light client that can do the following:

Comment (vthunder) [reply README.md:14]: Otherwise changes lgtm. Let's make sure we revisit all READMEs and docs on a regular basis and iteratively improve them.

**Comment by prabal-banerjee** [comment]
> > Minor changes in the intro is expected by end of week. Should we change it in this PR, you review and then we merge?
> 
> That works, the only obstacle is having access to the branch on your end

Would a rebase be better?

File: README.md (hunk 1 above)
Comment (DannyS03) [reply README.md]: Addressed

**Comment by DannyS03** [comment]
@prabal-banerjee - @kroos47 suggested to merge these changes first and rebase with the other branch

**Comment by vthunder** [comment]
One more thing that I'd like to change — this line is too complex:

> Data availability and computational integrity verified by avail light

And we removed this one which was clearer imo:

> Light client for Data Availability Blockchain of Polygon 💻

How about something like:

"Light client for the Polygon Avail blockchain" ?

Otherwise feel free to commit, looks good.

**Review by vthunder — APPROVED** [review]

**Pull request merged**